mail/dovecot/Portfile

```diff
@@ -1,110 +1,453 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
+PortGroup           github 1.0
 
+github.setup        dovecot core 2.3.9.2
 name                dovecot
+revision            0
 conflicts           dovecot2
-epoch               20060722
-version             1.2.17
-revision            2
-set branch          [join [lrange [split ${version} .] 0 1] .]
+epoch               20200128
+set core_version    ${version}
+
 categories          mail
-license             {MIT LGPL-2.1 BSD public-domain}
-maintainers         {jberry @jdberry} openmaintainer
+maintainers         nomaintainer
```

Comment:
Did jberry agree to this? Was a port abandoned ticket filed? If so, then please cite the Trac ticket otherwise reinstate the `maintainers` line as is.

Reply:
Whoops, somehow kept that line from the original version.

Reply:
@mf2k @jdberry @jmroot Please advise:

This line appears in the **current** `dovecot` version 1 portfile:
> `maintainers         {jberry @jdberry} openmaintainer`

This line appears in the **current** `dovecot2` version 2 portfile:
> `maintainers         nomaintainer`

Because we are replacing `dovecot` with `dovecot2`, wouldn't it make sense to use the same `maintainers` in the **new** `dovecot` as in the **current** `dovecot2`? Unless @jdberry explicitly agrees to be a maintainer for dovecot version 2.